rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -1015,4 +1062,31 @@ public boolean changeStartAt(JobId jobId, String startAt)
         }
         return isJobStartAtChanged;
     }
+
+    @Override
+    public TaskFlowJob getInitialJobContent(JobId jobId)
```

Comment:
If the getInitialJobContent logic is not needed right now and not used, I would vote to remove it (if it s simple to remove).